src/routes/v2/authenticatedSites/__tests__/Settings.spec.js

```diff
@@ -113,5 +113,19 @@ describe("Settings Router", () => {
       expect(mockSettingsService.retrieveSettingsFiles).toHaveBeenCalled()
       expect(mockSettingsService.updateSettingsFiles).toHaveBeenCalled()
     })
+    it("should allow for facebook pixel values of length 16", async () => {
```

Comment:
Hmm if we were to add test cases, should we not add test cases for both cases (ie 15 + 16?)?

Reply:
the default passing test is 15 digits alr